readme.txt

```diff
@@ -5,6 +5,9 @@ Tested up to: 5.1
 Stable tag: 1.0.1
 Requires at least: 4.4
 
+>**Important:**</br>
```

Comment:
Is formatting here incorrect?

Reply:
here > is used for blockquote and ** for making the important text bold.
This looks like this
https://share.getcloudapp.com/JruDkm12

Reply:
Actually we should remove everything from readme.md and just keep this message. readme.txt can have existing content and this extra string.

Reply:
So shall i remove all the content and keep this message only?

Reply:
Yes - in readme.md